scripts/dummy_data.py

```diff
@@ -1,81 +0,0 @@
-import glob,os,sys
```

Comment:
What are we doing to the dummy data here?

Reply:
@memo330179  You can ignore the dummy data. Jesson and I tried to merge my dummy data before I could make my commit but then later on decided against it.

Reply:
We need to be careful here looks like this completely wipes out the dummy data file, @Chauwaj could you add the file from development here?